src/afl-fuzz-mutators.c

```diff
@@ -192,7 +192,8 @@ void load_custom_mutator(afl_state_t *afl, const char *fn) {
 
   /* Initialize the custom mutator */
   if (afl->mutator->afl_custom_init)
-    afl->mutator->afl_custom_init(afl, rand_below(afl, 0xFFFFFFFF));
+    afl->mutator->data = afl->mutator->afl_custom_init(
+      afl, rand_below(afl, 0xFFFFFFFF));
```

Comment:
Oh great find!

Reply:
Also found some bugs in the custom mutator 😂
Try to solve them